kinto/config/__init__.py

```diff
@@ -67,6 +67,7 @@ def init(config_file, backend, cache_backend, host="127.0.0.1"):
 
     values["host"] = host
     values["secret"] = core_utils.random_bytes_hex(32)
+    values["bucket_id_salt"] = core_utils.random_urlsafe(32)
```

Comment:
Let's use the same function.

```suggestion
    values["bucket_id_salt"] = core_utils.random_bytes_hex(32)
```